### PR TITLE
Assignment 1 - Nolan Posey

### PR DESCRIPTION
**Pull request by Nolan-Posey**
https://brave-desert-0152b391e.3.azurestaticapps.net/

### CODE REVIEW

**Comment by StrangeRanger** [comment]
Nolan and I work together in a group. This is both of our submissions.

**Comment by StrangeRanger** [comment]
Nolan and I are in a group. This is our shared submission.

**Comment by Nolan-Posey** [comment]
@StrangeRanger 
@Nolan-Posey

**Review by slanante1997 — COMMENTED** [review]
Overall good job. I didn't find too much that could be changed mainly because I am also learning Vue and Vitest myself. I would now know what to recommend since I'm new to these. 

I noted a possible error in the checklist below, though that error may be my own misunderstanding of the requirements. Again, good job!

Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ❌
(I might be wrong on this requirement, but my understanding was we should have a button that links to the Assignment 1 github repo.)

Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest tests run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

**Comment by Nolan-Posey** [comment]
> Overall good job. I didn't find too much that could be changed mainly because I am also learning Vue and Vitest myself. I would not know what to recommend since I'm new to these.
> 
> I noted a possible error in the checklist below, though that error may be my own misunderstanding of the requirements. Again, good job!
> 
> Create a Vue app ✅
> 
> Uses TypeScript ✅ Add Vuetify ✅ Uses Vitest ✅ Create a simple page called game.vue ✅
> 
> Page should have a card with a heading that with a clever name for the game ✅ some default placeholder body text ✅ and a button to take you back to the home page ✅ there should be a button on the home page to access this page ❌ (I might be wrong on this requirement, but my understanding was we should have a button that links to the Assignment 1 github repo.)
> 
> Check all the code into your repo fork ✅
> 
> Create a static web app in Azure ✅
> 
> Create the container where your GitHub repo will deploy Link the site with your GitHub repo Update the CI/CD pipeline ✅
> 
> Pipeline runs on each check in Vitest tests run and pass ✅ Vue app builds ✅ Vue app deploys to Azure ✅ Azure site works ✅
> 
> Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

@slanante1997 I believe what it means when the assignment says "there should be a button on the home page to access this page ❌" its saying that there should be a button on the home page that links the user to the game page. The reason I believe this is because this bullet point is under the "Create a simple page called game.vue ✅" section. I will add a button that links to the "Assignment 1" page on github just incase.

Thank you for the review!

-Nolan Posey

File: wordle-web/src/views/Game.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <v-btn varient="outlined" rounded="xl" color="#5865f2" elevation="24" to="/about">
+          About
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </div>
+</template>
+
+<style>
+@media (min-width: 1024px) {
```
Comment (mmwoodfo) [inline wordle-web/src/views/Game.vue:27]: I don't think this is doing anything siginifcant.

File: wordle-web/src/views/HomeView.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  </main>
+  <v-card theme="dark">
+    <v-card-title> Home Page </v-card-title>
+
+    <v-card-subtitle> Welcome to the home page! </v-card-subtitle>
+
+    <v-card-actions>
+      <v-spacer />
+
+      <v-btn varient="outlined" rounded="xl" color="#5865f2" elevation="24" to="/Game"> Game </v-btn>
```
Comment (mmwoodfo) [inline wordle-web/src/views/HomeView.vue:13]: Looks like you have repeated code, this is a good use case for a component!

**Pull request closed without merging**